lib/matplotlib/offsetbox.py

```diff
@@ -369,7 +369,7 @@ def draw(self, renderer):
 
 
 class PackerBase(OffsetBox):
-    def __init__(self, pad=None, sep=None, width=None, height=None,
+    def __init__(self, pad=0, sep=0, width=None, height=None,
```

Comment:
```suggestion
    def __init__(self, pad=0., sep=0., width=None, height=None,
```
As it will be a bit more "correct" to pass a float as the default for a float argument, although no practical difference.